python/ray/tune/experiment/trial.py

```diff
@@ -908,6 +921,20 @@ def __setstate__(self, state):
         self.__dict__.update(state)
         self.stub = stub or getattr(self, "stub", False)
 
+        # Update CheckpointManager checkpoint paths with those found in the trial logdir
+        checkpoint_paths_df = TrainableUtil.get_checkpoints_paths(self.logdir)
```

Comment:
@krfricke This was the part that I was a bit iffy on. I opted to stick with this instead of splitting the path based on the old logdir, since the logdir has already been updated to the new one at this point.